.github/workflows/bot-hedera-mooncake.yml

```diff
@@ -0,0 +1,88 @@
+name: "[Bot] 'Hedera on Mooncake'"
+on:
+  push:
+    branches:
+      - feat/hedera-memo
```

Comment:
@henrily-ledger Just need to revert this back to feat/hedera, but we can do it after merging.